e2e/specs/biometrics.spec.ts

```diff
@@ -12,25 +12,38 @@ test.beforeEach(async ({ api }) => {
   visit = await startVisit(api, patient.uuid);
 });
 
-test('Record biometrics', async ({ page, api }) => {
+test('Record biometrics', async ({ page }) => {
   const biometricsPage = new BiometricsAndVitalsPage(page);
 
   await test.step('When I visit the vitals and biometrics page', async () => {
     await biometricsPage.goTo(patient.uuid);
   });
 
-  await test.step('And I click the `Record biometrics` link to launch the form', async () => {
+  await test.step('And I click on the `Record biometrics` link to launch the form', async () => {
     await biometricsPage.page.getByText(/record biometrics/i).click();
   });
 
-  await test.step('And then I fill the form', async () => {
+  await test.step('Then I should be redirected to the record vitals and biometrics form', async () => {
```

Comment:
Clicking the link launches the form in the workspace on the same page. We're not getting redirected to a different page.